Aptoide/appcoins-wallet-android
# [MOB-244] pull intercom to get notification

**What does this PR do?**

   This PR makes Intercom support available to all users.
   This PR changes the Intercom support icon to handle unread conversations

**Database changed?**

No

**Where should the reviewer start?**

- [ ] TransactionsActivity.java

**How should this be manually tested?**
Check if the icon changes when a new message is received

  Flow on how to test this or QA Tickets related to this use-case: [MOB-244](https://aptoide.atlassian.net/browse/MOB-244)

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-244](https://aptoide.atlassian.net/browse/MOB-244)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)
    Lottie and Intercom have been updated




**Code Review Checklist**
